### PR TITLE
JFG-863 add Dockerfile for jaas, fix webclient Dockerfile

### PR DESCRIPTION
**Pull request by Nmishin**

### CODE REVIEW

File: docker/compose/docker-compose.yml [hunk 1]
```diff
@@ -17,11 +18,23 @@ services:
     volumes:
       - ./files/my.cnf:/etc/mysql/conf.d/my.cnf
 
+  # Container with mysql base for jaas
+  jagger-jaas-mysql:
```
Comment (dlatnikov) [inline docker/compose/docker-compose.yml]: let's leave jaas-mysql

Comment (dlatnikov) [inline docker/compose/docker-compose.yml]: same below

File: docker/compose/docker-compose.yml [hunk 2]
```diff
@@ -17,11 +18,23 @@ services:
...
+      MYSQL_PASSWORD: password
+      MYSQL_DATABASE: jaasdb
+
   jagger-web-client:
-    image: griddynamics/jagger-web-client:shapshot
+    image: griddynamics/jagger-web-client:snapshot
     container_name: jagger-web-client
     ports:
-      - 8080:80
+      - 80:80
```
Comment (dlatnikov) [inline docker/compose/docker-compose.yml]: let's change default port

File: docker/compose/docker-compose.yml [hunk 3]
```diff
@@ -32,6 +45,24 @@ services:
...
       JWC_JDBC_PASS: password
-    # Link to containers in another service.
-    links:
-      - jagger-mysql
+
+  jagger-jaas:
+    image: griddynamics/jagger-jaas:snapshot
+    container_name: jagger-jaas
+    ports:
+      - 8080:8080
```
Comment (dlatnikov) [inline docker/compose/docker-compose.yml]: let's change default port

File: docker/jagger/jaas/run.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+   # Check if all important parameters exist
+if [ -z "$JAAS_HTTP_PORT" ] && [ -z "$JAAS_DB_DRIVER" ] && [ -z "$JAAS_DB_URL" ] && [ -z "$JAAS_DB_USER" ] && [ -z "$JAAS_DB_PASS" ] && [ -z "$JAAS_HIBERNATE_DIALECT"]; then
+    echo "env is empty, please specify parameters"
```
Comment (dlatnikov) [inline docker/jagger/jaas/run.sh]: what parameters?

**Comment by Nmishin** [comment]
new pull request 658 has been created

**Pull request closed without merging**